src/components/ContractCompiled/ContractCompiled.tsx

```diff
@@ -1,43 +1,27 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import "./ContractCompiled.css";
 
 interface IProps {
   contractName: string;
   bytecode: any;
```

Comment:
`Bytecode` typescript specification is here:
https://github.com/ethereum/remix-plugins-directory/blob/master/common-api/compiler/type/output.ts#L270